packages/formstr-sdk/src/formstr/formstr.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  SimplePool,
+  generatePrivateKey,
+  getEventHash,
+  getPublicKey,
+  getSignature,
+} from "nostr-tools";
+import { makeTag } from "../utils/utils";
+import { Field, FormSpec } from "../interfaces";
+
```

Comment:
```js
let relays = []

function setRelays(newRelays: string[]){
  relays = newRelays
}

```

Reply:
out of scope of this PR, will take this as a suggestion and open issue

Reply:
there is already an issue: https://github.com/abhay-raizada/nostr-forms/issues/11